apps/api/v2/src/ee/bookings/2024-08-13/controllers/e2e/team-bookings.e2e-spec.ts

```diff
@@ -205,6 +206,83 @@ describe("Bookings Endpoints 2024-08-13", () => {
 
       team1EventTypeId = team1EventType.id;
 
+      const phoneOnlyEventType = await eventTypesRepositoryFixture.createTeamEventType({
+        schedulingType: "ROUND_ROBIN",
+        team: {
+          connect: { id: team1.id },
+        },
+        title: "Phone Only Event Type",
+        slug: "phone-only-event-type",
+        length: 15,
+        assignAllTeamMembers: false,
+        hosts: {
+          connectOrCreate: [
+            {
+              where: {
+                userId_eventTypeId: {
+                  userId: teamUser.id,
+                  eventTypeId: team1EventTypeId,
+                },
+              },
+              create: {
+                userId: teamUser.id,
+                isFixed: true,
+              },
+            },
+          ],
+        },
+        bookingFields: [
+          {
+            name: "name",
+            type: "name",
+            label: "your name",
+            sources: [{ id: "default", type: "default", label: "Default" }],
+            variant: "fullName",
+            editable: "system",
+            required: true,
+            defaultLabel: "your_name",
+            variantsConfig: {
+              variants: {
+                fullName: {
+                  fields: [{ name: "fullName", type: "text", label: "your name", required: true }],
+                },
+              },
+            },
+          },
+          {
+            name: "email",
```

Comment:
It says "phoneOnlyEventTypeId" but here we also have email booking field. Shouldn't we remove email booking field because we have "attendeePhoneNumber".